### DISCUSSION TITLE
Where is the log?

### DISCUSSION THREAD

**Discussion by Nickduino** [IceWhaleTech/CasaOS#1887] [Q&A]
Brand new install, there are some apps that refuse to install but I have no idea what the error is as it is truncated.

![image](https://github.com/IceWhaleTech/CasaOS/assets/9784237/89a69fa4-ed7e-4160-97eb-ab45e8674ba2)

The cards aren't clickable and the logs on the top left show stuff entirely unrelated:
![image](https://github.com/IceWhaleTech/CasaOS/assets/9784237/2d6dc1cc-59ae-462c-8378-33a92f22429e)

To be clear: my issue is not the "failed to register layer...", it's the fact I don't know what the error is. I'm new to all this, could you help me find the error message in full?

**Comment by defencedog** [comment]
Same with me ...cannot copy paste any log

**Reply by AFK18** [reply]
> Same with me ...cannot copy paste any log

Is this in all logs? I'm trying to catch up with my admin and security and entangled passwords and all round meds... and learn how the operating system works on my Mac + Pc (which I want to run in sync sort of, but also independent) it's doing head in 

AFK

**Reply by AFK18** [reply]
Oh left message in email 

Do you blokes know good password and Authenticator and passkeys sync tips ?? Every thing entangled and I'm trying to set up a few things but need the tech and I'm obviously going cross eyed trying to disentangle 

Sorry if bother you 

Still don't understand purpose of gut hub but love tech more and more

Reminds me of working out XTree Pro in late 80s early 90s but it's come so far

I am obviously prehistoric brain here

**Reply by AFK18** [reply]
Oh left message in email 

Do you blokes know good password and Authenticator and passkeys sync tips ?? Every thing entangled and I'm trying to set up a few things but need the tech and I'm obviously going cross eyed trying to disentangle 

Sorry if bother you 

Still don't understand purpose of gut hub but love tech more and more

Reminds me of working out XTree Pro in late 80s early 90s but it's come so far

I am obviously prehistoric brain here

**Comment by ZDStudios** [comment]
When this error happens to me its a networking error so when im trying to access casaos using tailscale  i get the error then and sometimes at home so i just turn it off then on again and reconnect it internet connection and it works then
